### PR TITLE
Project init with skeleton, feat #1

### PR DESCRIPTION
**Pull request by ludwigjo**
Setup of project with the provided code skeleton and maven

### CODE REVIEW

File: .github/workflows/slack.yml [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+    branches: [ main ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+# A workflow run is made up of one or more jobs that can run sequentially or in parallel
+jobs:
+  # This workflow contains a single job called "build"
+  build:
+    environment: DECIDE
```
Comment (amandakrohn) [inline .github/workflows/slack.yml:17]: This should be changed to the name of the environment where the the secret is stored (in the last assignment Chiyi set up an environment named DECIDE where the secret was stored). If you named the environment DECIDE, then it is ok.

**Review by amandakrohn — COMMENTED** [review]
Also, should we add junit to the pom.xml (since unit tests are required), or is it not needed?

File: .github/workflows/slack.yml (hunk 1 above)
Comment (ludwigjo) [reply .github/workflows/slack.yml:17]: Alright good point! I'll fix that

**Comment by ludwigjo** [comment]
I can definitely add junit and create a test folder

**Comment by BrorSebastianSjovald** [comment]
Great job! I think it looks good now 👍

**Review by BrorSebastianSjovald — APPROVED** [review]

**Review by amandakrohn — APPROVED** [review]
Nice job! LGTM!

**Pull request merged**